src/screens/ExchangeModal.js

```diff
@@ -631,9 +631,26 @@ class ExchangeModal extends Component {
     }
   };
 
+  getNativeTag = field => get(field, '_nativeTag');
+
+  findNextFocused = () => {
```

Comment:
This is to handle the case when the lastFocusedInput actually isn't the right input to focus on next after navigating back from input currency selection. Most of the time it's fine, but for example, we need to handle the case when the input was ETH and the output wasn't selected, and the user selects ETH for the output (which automatically swaps the input to be null, and therefore the the lsatFocusedInput would no longer be valid)